docs/jvs-quick-setup.md

```diff
@@ -0,0 +1,156 @@
+# JVS Setup
```

Comment:
This is much better now! Thanks!

Reply:
Is there anything I need to change for this line?

Reply:
No